grid2op/tests/test_Opponent.py

```diff
@@ -44,9 +45,20 @@ def attack(self, observation, agent_action, env_action, budget, previous_fails):
         return attack
 
 
-class TestOpponentForConverter(BaseOpponent):
-    def __init__(self, action_space=None):
-        pass
+class ReconnectAgent(BaseAgent):
```

Comment:
This agent already exists no ?

Reply:
Oups forgot to remove it. But yes it does, as I pointed out to Loic :-)

Reply:
It's possible. I don't know. It was quick to write so I just did it